hulearn/experimental/interactive.py

```diff
@@ -139,11 +144,18 @@ def __init__(
         height=400,
         color=None,
         legend=True,
+        tooltip=None,
     ):
         self.uuid = str(uuid.uuid4())[:10]
         self.x = x
         self.y = y
-        self.plot = figure(width=width, height=height, title=f"{x} vs. {y}")
+        if tooltip is not None:
```

Comment:
I just tried out to see what happens when you pass in a spelling error. It makes sense to throw a proper `ValueError` here to prevent that we create a new `SingleInteractiveChart`. It will attach itself to the `InteractiveCharts`-object even if the tooltip fails and this is an unwanted side effect. 

![image](https://user-images.githubusercontent.com/1019791/107114620-80fe4600-6867-11eb-8236-745a5d6db712.png)

Could you also add a test that confirms that the `ValueError` is raised? You can add a `tests/test_interactive/test_interactive_chart.py` file to place it.